content/docs/tech-insights/track-datadog-slos/index.md

```diff
@@ -0,0 +1,123 @@
+
+In order to collect the Datadog SLO count from each component, the Datadog integration needs to be set up on Roadie. To do that, follow these steps:
+
+### Administration → Settings → Datadog
+
+Set the hostname of your Datadog app.
+
+### Administration → Settings → Secrets
+
+Set the `Datadog API Token` and `Datadog APP Token` in order for your Roadie instance to contact your Datadog app.
+
+| Key          | Token             |
+| ------------ | ----------------- |
+| DD_API_TOKEN | Datadog API Token |
+| DD_APP_TOKEN | Datadog APP Token |
+
+## Record the SLO count for each component
+
+In order to know which software has Datadog SLOs configured, we’re going to use the built-in Datadog Data Source in Roadie Tech Insights. It will run periodically and extract the SLO count from each component.
+
+These are the steps to set that up.
+
+1. Visit Tech Insights and click into the Data Sources tab. Search for the Datadog Data Source from the Data Sources list and select it.
+
+   ![Data Source Listing](./datasources_list.png)
+
+2. You may need to wait some time for the data source to collect all SLOs and Monitors from Datadog. It must contact the Datadog APIs for each component, which is captured by the already set filter.
+
+   ![Data Source Results](./datasource_results.png)
+
+## Visualize the distribution of SLOs count
+
+Our Data Source comes with a built-in visualization panel that lets us get an overview of the SLO counts present in each configured and annotated component. When viewing a Data Source, expand the “Facts visualization” section to see it.
+
+![Data Source Visualization](./datasource_graph.png)
+
+This chart tells us:
+
+1. 33% of the Components that this Data Source targets have 1 SLO defined and configured.
+2. 33% of the Components that this Data Source targets have 2 SLOs defined and configured.
+3. 33% of the Components that this Data Source targets have no SLOs configured.
+
+In the next section, we will create a Check that can show a pass or fail result to app dev teams to tell them if they are missing any component that doesn’t have an SLO defined.
+
+## Create a check that shows which software doesn’t have an SLO defined and configured
+
+Now that we can determine which components are using Dockerfiles, and we can extract the base image version from those files, let’s write a check to combine both of these properties.
+
+1. Visit Tech Insights and click into the Checks tab. Click the ADD CHECK button to create a new check.
+
+   ![Checks Listing](./checks_overview.png)
+
+2. Give the check a sensible name, like “Apps must have at least one Datadog SLO”, and a description like “Having Datadog SLOs configured for each component is important to measure the reliability and performance of services on a global scale.”
+
+   ![Create a Check](./create_check.png)
+
+3. In the Conditions section, we’re going to create a condition that compares against the Datadog Slo facts retrieved from the data source.
```

Comment:
I think the notion of a 'fact' is rather blurry as it wasn't introduced earlier. Might be useful to refer to the results of the Data Source as 'facts' in the previous section